packages/parser/src/repair.ts

```diff
@@ -19,12 +17,16 @@ function fixServers(
   server: OpenAPIV3_1.ReferenceObject | OpenAPIV3.ParameterObject | OpenAPIV3.ServerObject,
   path: string,
 ) {
-  // A erver URL starting with "/" tells that it is not an HTTP(s) URL.
+  // A server URL starting with "/" tells that it is not an HTTP(s) URL.
   if (server && 'url' in server && server.url && server.url.startsWith('/')) {
-    const inUrl = url.parse(path);
```

Comment:
The `url.parse` method that we were importing from this was literally just a wrapper for `new URL()`.